packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.js

```diff
@@ -31,7 +31,7 @@ export default ({ coreSagas }) => {
         to: to,
         coin: 'BTC',
         amount: amount,
-        message: message,
+        description: message,
```

Comment:
Yeah it's a little confusing because

- ETH: description (https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/modals/SendEther/FirstStep/template.success.js#L67)
- BTC: description (https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/modals/SendBitcoin/FirstStep/template.success.js#L140)
- BCH: message (https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/modals/SendBch/FirstStep/template.success.js#L83)

Can you take some time to make sure we are using `description` consistently for btc/bch/eth in:
1. the modals/send first step templates
2. 1. the modals/send second step selectors
3. the data/components/send[Btc|Bch|Eth] sagas.js

Reply:
This has to be 'description' for the form to initialize with it. I guess I could change that in the form so that they are both message. It was inconsistent before and stopping the message data from going to the form.